piqueserver/game_modes/infiltration.py

```diff
@@ -78,8 +78,8 @@ def score(self):
         intel_capture.winning = winning
         self.protocol.send_contained(intel_capture, save=True)
         if winning:
-            self.team.initialize()
-            self.team.other.initialize()
+            self.team = self.team(*self.team.get_init_values())
+            self.team.other = self.team(*self.team.other.get_init_values())
```

Comment:
This will probably break the game mode. Should also make it so self.team.other.other = self.team

Reply:
Added